neqo-qpack/src/lib.rs

```diff
@@ -27,6 +27,8 @@ mod static_table;
 pub mod stats;
 mod table;
 
+pub use decoder::QPackDecoder;
```

Comment:
If you are going to expose this this way, then you should also expose the encoder.